pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -139,14 +143,71 @@ func setupC2C(ctx context.Context, t test.Test, c cluster.Cluster) (*c2cSetup, f
 		dst: destTenantInfo}, cleanup
 }
 
+type streamingWorkload interface {
+	// name returns the name of the workload
+	name() string
+
+	// sourceInitCmd returns a command that will populate the src cluster with data before the
+	// replication stream begins
+	sourceInitCmd(pgURL string) string
+
+	// sourceRunCmd returns a command that will run a workload for the given duration on the src
+	// cluster during the replication stream.
+	sourceRunCmd(pgURL string, duration time.Duration) string
+}
+
+type replicateTPCC struct {
+	warehouses int
+}
+
+func (tpcc replicateTPCC) name() string {
+	return fmt.Sprintf("tpcc/warehouses=%d", tpcc.warehouses)
+}
+
+func (tpcc replicateTPCC) sourceInitCmd(pgURL string) string {
+	return fmt.Sprintf(`./workload init tpcc --data-loader import --warehouses %d '%s'`,
+		tpcc.warehouses, pgURL)
```

Comment:
about a minute or so into this initial import, the tpcc init import jobs slow to a crawl-- occasionally an addsstable request does go through. I checked the goroutines on the src cluster kv nodes and none seem to be processing addSSTable requests, so I think the bottleneck is on the tenant. Here's my investigation thus far, and I'm wondering if you have pointers of where to look next:

I noticed that on the dbconsole, that the `cr.node.tenant.consumption.kv_request_units` metric steadily climbed and then capped at 10m. I wonder if the tenant's requests are getting throttled, as serverless has a cap of [10m RU ](https://www.cockroachlabs.com/docs/cockroachcloud/architecture.html#baseline) of burst capacity. I tried increasing the `kv.tenant_rate_limiter.burst_limit_seconds` and the `kv.tenant_rate_limiter.rate_limit` on a kv node and that didn't seem to help. Perhaps I'm barking up the wrong tree.

The logs from the src tenant indicate that a flurry of addsstable requests continue to be retried, though I couldn't ascertain why.

Reply:
upping RUs fixed this problem.